### ISSUE TITLE
Unarchive LiveSDK.framework

### ISSUE THREAD

**Issue by constantine-fry** [liveservices/LiveSDK-for-iOS#26]
Please include unarchived LiveSDK.framework in repository so everybody can include it as submodule.

**Comment by wangdxnum1** [comment]
Yes,I also hope include the LiveSDK.framework.I build by my self,but I find that the back image's path is not right ,in LiveAuthHelper,+ (UIImage *) getBackButtonImage. the resource path is "LiveSDK.framework/Resources/backArrow_black",but In my LiveSDK.framework,there is no folder named "Resources",so the image can not be found.
